scene/main/viewport.cpp

```diff
@@ -462,6 +462,10 @@ void Viewport::_notification(int p_what) {
 			RenderingServer::get_singleton()->viewport_set_parent_viewport(viewport, RID());
 		} break;
 		case NOTIFICATION_INTERNAL_PHYSICS_PROCESS: {
+			if (!get_tree()) {
+				return;
+			}
+			
```

Comment:
There's trailing whitespace on this line that should be removed (you should do it by editing this commit with `git commit --amend`, and then force push to update the PR, see [rebase docs](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase)).

Reply:
done